### PR TITLE
Submission models

### PR DESCRIPTION
**Pull request by lennym**
This needs testing end-to-end with file uploads and icasework, but this is a decent enough first candidate.

Each app has `apps/**/models/submission.js` which returns mapped fields for that app. Default stuff and auth is handled in `apps/common/models/i-casework.js`.

To test this, you'll need icasework auth params as environment variables. Hit me up for these.

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -44,36 +44,36 @@ pipeline:
...
+  # test_acceptance:
+  #   image: docker:1.11
+  #   environment:
+  #     - DOCKER_HOST=tcp://127.0.0.1:2375
+  #   commands:
+  #     - docker run -e NODE_ENV=ci -e DISABLE_CSP=true -d --net host firearms
+  #     - docker run --rm --net host acceptance
+  #   when:
+  #     branch: master
+  #     event: push
```
Comment (sulthan-ahmed) [inline .drone.yml]: any reason why these are commented out rather than not deleted?

File: apps/common/views/partials/confirmation-alert.html [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<div class="alert-complete" role="alert">
+  <div class="alert-message">
+    <h1>{{#t}}pages.confirmation.alert{{/t}}</h1>
+    {{#values.caseid}}
+    <p>{{#t}}pages.confirmation.reference{{/t}}</p>
+    {{/values.caseid}}
+  </div>
+</div>
```
Comment (sulthan-ahmed) [inline apps/common/views/partials/confirmation-alert.html]: missing EOF line

File: .drone.yml (hunk 1 above)
Comment (easternbloc) [inline .drone.yml]: they will return in the next few days so i'm just commenting them out for now

Comment (lennym) [inline .drone.yml]: Bet they don't

Comment (sulthan-ahmed) [inline .drone.yml]: that's fine

**Pull request merged**